stdlib/public/Platform/glibc.modulemap.gyb

```diff
@@ -36,6 +36,12 @@ module SwiftGlibc [system] {
   link "execinfo"
 % end
 
+% if CMAKE_SDK in ["FREEBSD"]:
+  module cdefs {
+    header "sys/cdefs.h"
```

Comment:
This probably needs `${GLIBC_INCLUDE_PATH}` ? 
Otherwise I get

```
/usr/home/davide/build/Ninja-RelWithDebInfoAssert/swift-freebsd-x86_64/lib/swift/freebsd/x86_64/glibc.modulemap:36:12: error: header 'sys/cdefs.h' not found
    header "sys/cdefs.h"
```

Reply:
Yeah, oops, it does.